.env

```
@@ -0,0 +1,9 @@
+NEXT_PUBLIC_APP_URL=http://localhost:3000
+NEXT_PUBLIC_FIREBASE_API_KEY=AIzaSyCw76TpWd-MQNZqsC60FS71lhuHks8YJDw
+FIREBASE_PROJECT_ID=matrim-flowboard
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The FIREBASE_PROJECT_ID key should go before the NEXT_PUBLIC_APP_URL key

(UnorderedKey)